config/set/nette-31.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Composer\Rector\ChangePackageVersionRector;
+use Rector\Composer\Rector\RemovePackageRector;
+use Rector\Composer\ValueObject\PackageAndVersion;
+use Rector\Renaming\Rector\MethodCall\RenameMethodRector;
+use Rector\Renaming\Rector\Name\RenameClassRector;
+use Rector\Renaming\Rector\StaticCall\RenameStaticMethodRector;
+use Rector\Renaming\ValueObject\MethodCallRename;
+use Rector\Renaming\ValueObject\RenameStaticMethod;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\SymfonyPhpConfig\ValueObjectInliner;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->set(RenameClassRector::class)->call('configure', [[
+        RenameClassRector::OLD_TO_NEW_CLASSES => [
+            // https://github.com/nette/application/compare/v3.0.7...v3.1.0
+            'Nette\Application\IRouter' => 'Nette\Routing\Router',  // TODO not sure about this, it is not simple rename, Nette\Routing\Router already exists in nette/routing
```

Comment:
Rector should handle it. Did you came across any bug? 
If so, we'll need a test fixture to fix it.

Reply:
I thought it will change also usage of Nette\Application\IRouter in use and there could be two same uses:
```php
<?php

use Nette\Application\IRouter;
use Nette\Routing\Router;

class RouterTest
{
    public static function createRouter()
    {
        $router = new Router();
        $router->addRoute('a', 'b', IRouter::ONE_WAY);
        return $router;
    }
}
```
but it only replace usage in code, so no problem :)